### PR TITLE
add named export to improve `"module": "nodenext"` compatibility

### PR DESCRIPTION
**Pull request by wight554**
* see https://github.com/microsoft/TypeScript/issues/48845

### CODE REVIEW

**Comment by wight554** [comment]
Any progress on this one @JoviDeCroock ?
It fixes this issue:
image

File: src/index.ts [hunk 1]
```diff
@@ -187,3 +187,5 @@ export default function preactPlugin({
 		prefresh({ include, exclude }),
 	];
 }
+
+export { preactPlugin as default, preactPlugin as preact };
```
Comment (JoviDeCroock) [inline src/index.ts:191]: You are addina new export here 😅

Comment (JoviDeCroock) [inline src/index.ts:191]:
```suggestion
export default preactPlugin;
export { preactPlugin as preact };
```

**Review by JoviDeCroock — APPROVED** [review]

**Comment by kidonng** [comment]
Can this change be released? Doesn't seem like a breaking change.

**Comment by marvinhagemeister** [comment]
No, not a particular reason, this somehow went through the cracks between all the notifications.

**Pull request merged**